.github/workflows/azure-deploy.yml

```
@@ -0,0 +1,207 @@
+      AZURE_RESOURCE_GROUP: ${{ vars.AZURE_RESOURCE_GROUP }}
+      AZURE_ACR_NAME: ${{ vars.AZURE_ACR_NAME }}
+      AZURE_WEB_APP_NAME: ${{ vars.AZURE_WEB_APP_NAME }}
+      AZURE_MIGRATE_JOB_NAME: ${{ vars.AZURE_MIGRATE_JOB_NAME }}
+      AZURE_QUEUE_JOB_NAME: ${{ vars.AZURE_QUEUE_JOB_NAME }}
+      AZURE_RESTORE_JOB_NAME: ${{ vars.AZURE_RESTORE_JOB_NAME }}
+      AZURE_PROVISION_JOB_NAME: ${{ vars.AZURE_PROVISION_JOB_NAME }}
+      AZURE_SCHED_HOURLY_JOB_NAME: ${{ vars.AZURE_SCHED_HOURLY_JOB_NAME }}
+      AZURE_SCHED_DAILY_JOB_NAME: ${{ vars.AZURE_SCHED_DAILY_JOB_NAME }}
+      AZURE_SCHED_WEEKLY_JOB_NAME: ${{ vars.AZURE_SCHED_WEEKLY_JOB_NAME }}
+      AZURE_SCHED_NIGHTLY_JOB_NAME: ${{ vars.AZURE_SCHED_NIGHTLY_JOB_NAME }}
+      SNAPSHOT_DIR: ${{ runner.temp }}/kmp-aca-pre-cutover
```

actionlint (1.7.12)

[error] context "runner" is not allowed here. available contexts are "github", "inputs", "matrix", "needs", "secrets", "strategy", "vars". see https://docs.github.com/en/actions/learn-github-actions/contexts#context-availability for more details

(expression)